raiden/network/transport.py

```diff
@@ -81,6 +87,9 @@ def __init__(
     def receive(self, data, host_port):  # pylint: disable=unused-argument
         try:
             self.protocol.receive(data)
+        except InvalidProtocolMessage as e:
+            log.warning('Can\'t decode: {} (data={}, len={})'.format(str(e), data, len(data)))
```

Comment:
Please use double quotes here. See the [guidelines](https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md)